src/cascadia/TerminalSettingsEditor/Appearances.idl

```diff
@@ -30,7 +32,7 @@ namespace Microsoft.Terminal.Settings.Editor
         Boolean UseDesktopBGImage;
         Boolean BackgroundImageSettingsVisible { get; };
 
-        Windows.Foundation.Collections.IMapView<String, Microsoft.Terminal.Settings.Model.ColorScheme> Schemes;
+        ColorSchemesPageViewModel SchemesPageVM;
```

Comment:
Hmmm.. this seems like maybe the wrong level to me. I would expect that we would get a list of `ColorSchemeViewModel`s here instead of access to the entire color schemes page.

Having access to the color schemes page's viewmodel leaks a wider-scoped responsibility over to Appearances than just having the color schemes.

At the same time, this lets you consolidate down to there only being one main list of color scheme viewmodel objects...

Reply:
That makes sense, fixed